### PR TITLE
Part of #18714 for 2 files in ui\components\component-library\avatar-icon

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a part of the issue #18714 for 2 files in 
_ui\components\component-library\avatar-icon_
1. **ui\components\component-library\avatar-icon\avatar-icon.js**
2. **ui\components\component-library\avatar-icon\avatar-icon.stories.js**

### Before

Screenshot (172)
Screenshot (174)
Screenshot (175)

### After

Screenshot (177)

Screenshot (176)
Screenshot (173)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall , 

Please look into this PR.
All tests have passed.

File: ui/components/component-library/avatar-icon/avatar-icon.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import classnames from 'classnames';
 import {
   BorderColor,
   Size,
-  DISPLAY,
+  // DISPLAY,
```
Comment (darkwing) [inline ui/components/component-library/avatar-icon/avatar-icon.js:7]: Looks like this comment needs to be removed.

**Review by darkwing — CHANGES_REQUESTED** [review]
Just needs a small update -- thank you!

File: ui/components/component-library/avatar-icon/avatar-icon.js (hunk 1 above)
Comment (PrgrmrHarshShukla) [reply ui/components/component-library/avatar-icon/avatar-icon.js]: 👍
Done Sir.

File: ui/components/component-library/avatar-icon/avatar-icon.stories.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   Size,
-  DISPLAY,
   AlignItems,
   BackgroundColor,
   IconColor,
   Color,
+  Display,
 } from '../../../helpers/constants/design-system';
 
 import Box from '../../ui/box/box';
```
Comment (georgewrmarshall) [inline ui/components/component-library/avatar-icon/avatar-icon.stories.js:11]: Could you update this storybook file to use the `Box` component from the component-library please?

**Review by georgewrmarshall — COMMENTED** [review]
One suggestion

File: ui/components/component-library/avatar-icon/avatar-icon.stories.js (hunk 2 above)
Comment (PrgrmrHarshShukla) [reply ui/components/component-library/avatar-icon/avatar-icon.stories.js:11]: Sure @georgewrmarshall ,
Will do.

Comment (PrgrmrHarshShukla) [reply ui/components/component-library/avatar-icon/avatar-icon.stories.js:11]: Hey @georgewrmarshall ,
Probably, I am doing some mistake in trying to get the `Box` component from the `component-library`
OR, probably it is imported and exported in some other way.

Will try to figure out and report to you soon.

Comment (PrgrmrHarshShukla) [reply ui/components/component-library/avatar-icon/avatar-icon.stories.js]: Hey @georgewrmarshall ,
Tried to do that in the last commit.

Please look into it.

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Review by darkwing — APPROVED** [review]

**Pull request merged**